src/lightkurve/targetpixelfile.py

```diff
@@ -2045,7 +2051,60 @@ def plot_pixels(
 
         return ax
 
+    @property
+    def skycatalog(self):
```

Comment:
A style comment: exposing the list of neighboring TICs/KICs/EPICs via a property `tpf.skycatalog` could be somewhat misleading from programming point of view. 

Conventionally, data exposed via properties tend to be in-memory or easily computed. The `tpf.skycatalog` here is much more involved.

An alternative is to provide it as a method `tpf.query_skycatalog(...)` with parameters' default values do what the properties currently does.

Reply:
@christinahedges what do you think with regards to this?

Reply:
For the use cases for supporting contamination calculation / PRF, etc,
1. it might make sense to let users easily customize the catalog to 
  a. include stellar companion identified in follow up, and/or
  b. exclude duplicates, artifacts, etc.
One way to do it is through a method tpf.query_skycatalog(...), something like the following:
```python
# add a stellar companion identified in follow up
cat = tpf.query_skycatalog(additions=[
    dict(id="stellar companion 1", sep=0.3*u.arcsec, angle=Angle(131.3 * u.deg), delta_mag=1),
])

# TIC 237184773 is split to 1715469662, 1715469667
cat = tpf.query_skycatalog(subtractions=[237184773])
```

2. Are we concerned about the cases a bright star that is just outside of the tpf, but it significantly affects many pixels inside the tpf?
This is probably too much of an edge case. I suppose users can workaround it somewhat easily if they can add additional stars, say, via `tpf.query_skycatalog(additions=...)`.